sources/install/package_base.sh

```diff
@@ -173,6 +173,27 @@ function install_ultimate_vimrc() {
     sh ~/.vim_runtime/install_awesome_vimrc.sh
 }
 
+function install_neovim() {
+    if [[ $(uname -m) = 'x86_64' ]]
+    then
+        curl -LO https://github.com/neovim/neovim/releases/latest/download/nvim.appimage
+        chmod u+x nvim.appimage
+        ./nvim.appimage --appimage-extract
+        cp -r squashfs-root/usr/ /opt/tools/nvim
+        rm -rf squashfs-root nvim.appimage
+    elif [[ $(uname -m) = 'aarch64' ]]
+    then
+        git clone https://github.com/neovim/neovim.git
+        cd neovim
+        make CMAKE_BUILD_TYPE=RelWithDebInfo
+        make install
+        cd ..
+        rm -rf neovim
```

Comment:
I prefer when the paths are precise : `./neovim`

Reply:
https://github.com/ThePorgs/Exegol-images/pull/204/commits/80620300a21aafd6b778170e1dd71a4aa8b745f4